### DISCUSSION TITLE
Difference between standard and J1939 filtering

### DISCUSSION THREAD

**Discussion by bvernham** [collin80/SavvyCAN#338] [Q&A]
What specifically is the difference in masking between standard and J1939 filter masking?

Does the masking ignore both Source and Destination addresses?

Thanks

Bruce

**Comment by collin80** [comment]
That's the idea. Unfortunately, it isn't quite all hooked up yet so you'll get a masking system reported but it doesn't entirely actually do what it says. At the moment I believe the labeling works with the matching criteria but actual interpretation of signals isn't using the proper masking so things are still not really working properly for J1939 and GMLAN signals.

**Comment by bvernham** [comment]
Thanks for the quick reply, you are doing great work.

BTW have you looked at doing anything with the Teensy4.1?

That thing is a beast!  3 CAN channels/64 MB each/Tons of RAM and flash.

10 faster than the ESP32.  32 and 64 bit FP using and so on.

Thanks

Bruce
